packages/editor/src/store/defaults.js

```diff
@@ -27,5 +27,4 @@ export const EDITOR_SETTINGS_DEFAULTS = {
 	richEditingEnabled: true,
 	codeEditingEnabled: true,
 	enableCustomFields: undefined,
-	supportsLayout: true,
```

Comment:
Any reasons for the changes to the "default" values? What impact this could have?

Reply:
This value is wrong for the mobile editor. It should be inherited from default block editor settings (the `...SETTINGS_DEFAULTS` spread a few lines above) where it's `supportsValue: false` for the mobile editor.

Until now it didn't matter because nobody read the `supportsValue` from the editor settings. But now editor settings are synchronized to block editor settings, and the bad value is propagated there. And it affects whether the "wide" and "full" block alignment options are displayed.